bin/cron/update_pledge_map

```diff
@@ -43,41 +57,8 @@ begin
 
       next unless submission[:school_id]
       row_id = "CENSUS_SUBMISSIONS-#{submission[:census_submissions_id]}"
-      row = [row_id, submission[:name], submission[:zip], submission[:city], type]
-      csv << row
-    end
-
-    DB_READONLY[:forms].where("kind in (#{CENSUS_FORM_KINDS})").paged_each(rows_per_fetch: 10_000) do |form|
-      # Skip any rows that were migrated to CENSUS_SUBMISSIONS since those were already processed above.
-      form_map = DASHBOARD_DB[:census_submission_form_maps].where(form_id: form[:id])
-      next unless form_map.empty?
-      data = JSON.parse(form[:data])
-
-      # Was a pledge made?
-      pledged = ["on", "1"].any? {|s| s.casecmp(data["pledge_b"].to_s).zero?}
-
-      # Does the school teach some kind of CS?
-      teaches_cs = ["some", "all"].any? {|s| s.casecmp(data["ten_hours_s"].to_s).zero?} ||
-          ["some", "all"].any? {|s| s.casecmp(data["twenty_hours_s"].to_s).zero?}
-
-      # Only add to the map if pledge checkbox is checked or the school teaches some form of CS.
-      next unless pledged || teaches_cs
```

Comment:
We don't need this check anymore?

Reply:
That is the code that was reading data from FORMS. There is an equivalent check in the code that reads from CENSUS_SUBMISSIONS.